packages/form-js-editor/src/features/properties-panel/entries/RepeatableEntry.js

```diff
@@ -0,0 +1,52 @@
+import { zeroPositiveIntegerEntryFactory, simpleBoolEntryFactory } from './factories';
+
+export default function RepeatableEntry(props) {
+  const {
+    field,
+    getService
+  } = props;
+
+  const {
+    type
+  } = field;
+
+  const formFieldDefinition = getService('formFields').get(type);
+
+  if (!formFieldDefinition || !formFieldDefinition.config.repeatable) {
+    return [];
+  }
+
+  const entries = [
+    zeroPositiveIntegerEntryFactory({
+      id: 'defaultRepetitions',
+      path: [ 'defaultRepetitions' ],
+      label: 'Default number of items',
+      props
+    }),
+    simpleBoolEntryFactory({
+      id: 'allowAddRemove',
+      path: [ 'allowAddRemove' ],
+      label: 'Allow add/delete items',
+      props
+    }),
+    simpleBoolEntryFactory({
+      id: 'disableCollapse',
+      path: [ 'disableCollapse' ],
+      label: 'Disable collapse',
+      props
+    })
+  ];
+
+  if (!field.disableCollapse) {
+    const nonCollapseItemsEntry = zeroPositiveIntegerEntryFactory({
+      id: 'nonCollapsedItems',
+      path: [ 'nonCollapsedItems' ],
```

Comment:
We need to update the JSON schema that has moved to this project. Let's open an issue for that as a follow-up!

`dynamiclist` type and `verticalAlignment` should already be present.

Reply:
https://github.com/bpmn-io/form-js/issues/907